vnext/JSI/Shared/ChakraRuntime.cpp

```diff
@@ -607,284 +616,202 @@ facebook::jsi::Runtime::ScopeState *ChakraRuntime::pushScope() {
 
 void ChakraRuntime::popScope(Runtime::ScopeState *state) {
   assert(state == nullptr);
-  checkException(JsCollectGarbage(m_runtime), "JsCollectGarbage");
+  ThrowUponJsError(JsCollectGarbage(m_runtime), "JsCollectGarbage");
 }
 
 bool ChakraRuntime::strictEquals(
```

Comment:
>ChakraRuntime::strictEquals( [](start = 5, length = 28)

Are the 3 overloads even needed? String/Object/Symbol all derive from Pointer.

Reply:
I don't think they are needed either, but these three overloads are virtual functions we inherit from jsi::Runtime. We should probably see if we can push a change to Facebook to merge them.

---
In reply to: [336276579](https://github.com/microsoft/react-native-windows/pull/3445#discussion_r336276579) [](ancestors = 336276579)